### DISCUSSION TITLE
Welcome to my-notes Discussions!

### DISCUSSION THREAD

**Discussion by lvr1997** [lvr1997/my-notes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sslangss** [comment]
想请教一下turn.js翻页插件的二改问题

**Reply by lvr1997** [reply]
请问你是遇到什么问题了呢？
